src/app/modules/shared-components/components/input/input.component.scss

```diff
@@ -63,9 +63,25 @@
 
     &::placeholder {
       font-style: italic;
-      color: lightgray;
+      color: rgb(118, 118, 121);
       font-size: 1.2rem;
     }
+
+
+  }
+
+  .clear-button {
```

Comment:
could you make it grey as the text and the left icon?

Reply:
Done